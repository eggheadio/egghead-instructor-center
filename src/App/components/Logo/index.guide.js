import React from 'react'
import {storiesOf} from '@kadira/storybook'
import Logo from './index'

storiesOf('Logo', module)
  .add('Default', () => (
    <Logo />
  ))
