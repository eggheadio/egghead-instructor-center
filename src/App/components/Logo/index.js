import React from 'react'
import logo from './logo.svg'

const Logo = () => (
  <img
    src={logo}
    alt="egghead.io logo"
  />
)

export default Logo
