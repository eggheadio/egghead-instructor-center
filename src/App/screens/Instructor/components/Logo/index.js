import React from 'react'
import logo from './logo.svg'

export default () => (
  <img
    src={logo}
    alt="egghead.io logo"
  />
)
