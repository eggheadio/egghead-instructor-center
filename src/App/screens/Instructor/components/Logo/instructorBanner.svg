<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1636.38 184.71"><defs><style>.cls-1{fill:#fffffb;}.cls-2{fill:#fcbf53;}.cls-3{fill:url(#linear-gradient);}.cls-4{fill:#f1edeb;}.cls-5{fill:#9399a1;opacity:0.3;}.cls-6{fill:#e97c43;}.cls-7{fill:#ec8030;}.cls-8{fill:#f09f4f;}.cls-9{fill:#cbcc5f;}.cls-10{fill:#fbd170;}</style><linearGradient id="linear-gradient" x1="9734.22" y1="30.38" x2="9734.22" y2="65.46" gradientTransform="matrix(-1, 0, 0, 1, 9863.3, 0)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fcd06a"/><stop offset="1" stop-color="#ee8d4c"/></linearGradient></defs><title>Instructor-Title-On-Dark</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M273.55,128.09a33.07,33.07,0,0,1-4.59,9.74,33.65,33.65,0,0,1-7.65,8,36.81,36.81,0,0,1-10.55,5.4,41.81,41.81,0,0,1-13.29,2,43.55,43.55,0,0,1-15.62-2.82,37.8,37.8,0,0,1-13-8.29,39.75,39.75,0,0,1-8.94-13.37,45.93,45.93,0,0,1-3.3-17.88,44.4,44.4,0,0,1,3.22-17.23,40.14,40.14,0,0,1,8.62-13.13,38.71,38.71,0,0,1,12.48-8.38,37.46,37.46,0,0,1,14.66-3q18.52,0,28.83,11t10.31,30.12q0,2.26-.16,4.27t-.16,2.33H217.67a17.07,17.07,0,0,0,1.85,7.17,19.92,19.92,0,0,0,4.35,5.72,19.61,19.61,0,0,0,6.2,3.79,20.79,20.79,0,0,0,7.57,1.37q7.57,0,11.84-3.38a19.26,19.26,0,0,0,6.2-8.7Zm-20-25.93a19.41,19.41,0,0,0-1.21-5.72,13.8,13.8,0,0,0-3.22-5,16.63,16.63,0,0,0-5.48-3.54,20.33,20.33,0,0,0-7.81-1.37A17.91,17.91,0,0,0,228.46,88,18.24,18.24,0,0,0,223,91.7a16.4,16.4,0,0,0-3.46,5,15.78,15.78,0,0,0-1.37,5.47Z"/><path class="cls-1" d="M302.26,150a17.78,17.78,0,0,0,6,10.79Q313,165,320.78,165q10.14,0,15.62-5.23t5.48-16.83v-5.8a21.8,21.8,0,0,1-7.89,7q-5.48,3.14-14.17,3.14a36.79,36.79,0,0,1-14.66-2.9,34.94,34.94,0,0,1-11.68-8.05,37.92,37.92,0,0,1-7.73-12.16,42.65,42.65,0,0,1-.08-30.28,39.5,39.5,0,0,1,7.57-12.32,35,35,0,0,1,11.59-8.37,35.91,35.91,0,0,1,15-3.06q9.5,0,15,3.38t7.57,7.41V71.57H363v70.7a52.48,52.48,0,0,1-2.34,15.7,35,35,0,0,1-20.21,22.39q-7.89,3.46-19,3.46a45,45,0,0,1-14.9-2.33,39.94,39.94,0,0,1-11.68-6.28A32.63,32.63,0,0,1,286.8,166a29.73,29.73,0,0,1-3.86-10.87Zm21.26-21.1A17.78,17.78,0,0,0,337,123.42q5.23-5.47,5.23-14.66t-5.48-14.57a18.14,18.14,0,0,0-13.21-5.4A20.19,20.19,0,0,0,316,90.17a17.08,17.08,0,0,0-6,4,18.87,18.87,0,0,0-4,6.36,22.35,22.35,0,0,0-1.45,8.21q0,9.34,5.31,14.74A18.38,18.38,0,0,0,323.52,128.9Z"/><path class="cls-1" d="M392.32,150a17.78,17.78,0,0,0,6,10.79q4.83,4.18,12.56,4.19,10.14,0,15.62-5.23t5.48-16.83v-5.8a21.8,21.8,0,0,1-7.89,7q-5.48,3.14-14.17,3.14a36.79,36.79,0,0,1-14.66-2.9,34.94,34.94,0,0,1-11.68-8.05,37.92,37.92,0,0,1-7.73-12.16,42.65,42.65,0,0,1-.08-30.28,39.5,39.5,0,0,1,7.57-12.32,35,35,0,0,1,11.59-8.37,35.91,35.91,0,0,1,15-3.06q9.5,0,15,3.38t7.57,7.41V71.57H453v70.7A52.48,52.48,0,0,1,450.7,158a35,35,0,0,1-20.21,22.39q-7.89,3.46-19,3.46a45,45,0,0,1-14.9-2.33,39.94,39.94,0,0,1-11.68-6.28,32.63,32.63,0,0,1-8.05-9.18A29.73,29.73,0,0,1,373,155.15Zm21.26-21.1A17.78,17.78,0,0,0,427,123.42q5.23-5.47,5.23-14.66t-5.48-14.57a18.14,18.14,0,0,0-13.21-5.4,20.19,20.19,0,0,0-7.49,1.37,17.08,17.08,0,0,0-6,4,18.87,18.87,0,0,0-4,6.36,22.35,22.35,0,0,0-1.45,8.21q0,9.34,5.31,14.74A18.38,18.38,0,0,0,413.58,128.9Z"/><path class="cls-1" d="M485,150.8H465.79V34.2H485v44a21.34,21.34,0,0,1,9.82-6.68,36.13,36.13,0,0,1,11.6-2A31.79,31.79,0,0,1,519.52,72a26.06,26.06,0,0,1,9.18,6.68,27.2,27.2,0,0,1,5.4,10,41.68,41.68,0,0,1,1.77,12.24V150.8H514.45V104.58A18,18,0,0,0,511,93.31q-3.46-4.51-11.19-4.51-6.77,0-10.63,4.43a17.53,17.53,0,0,0-4.19,11Z"/><path class="cls-1" d="M621.68,128.09a33.07,33.07,0,0,1-4.59,9.74,33.65,33.65,0,0,1-7.65,8,36.81,36.81,0,0,1-10.55,5.4,41.81,41.81,0,0,1-13.29,2A43.55,43.55,0,0,1,570,150.4a37.8,37.8,0,0,1-13-8.29A39.75,39.75,0,0,1,548,128.74a45.93,45.93,0,0,1-3.3-17.88,44.4,44.4,0,0,1,3.22-17.23,40.14,40.14,0,0,1,8.62-13.13A38.71,38.71,0,0,1,569,72.13a37.46,37.46,0,0,1,14.66-3q18.52,0,28.83,11t10.31,30.12q0,2.26-.16,4.27t-.16,2.33H565.79a17.07,17.07,0,0,0,1.85,7.17A19.92,19.92,0,0,0,572,129.7a19.61,19.61,0,0,0,6.2,3.79,20.79,20.79,0,0,0,7.57,1.37q7.57,0,11.84-3.38a19.26,19.26,0,0,0,6.2-8.7Zm-20-25.93a19.41,19.41,0,0,0-1.21-5.72,13.8,13.8,0,0,0-3.22-5,16.63,16.63,0,0,0-5.48-3.54A20.33,20.33,0,0,0,584,86.54,17.91,17.91,0,0,0,576.58,88a18.24,18.24,0,0,0-5.48,3.71,16.4,16.4,0,0,0-3.46,5,15.78,15.78,0,0,0-1.37,5.47Z"/><path class="cls-1" d="M629.92,129.22a21.94,21.94,0,0,1,6.92-16.59,24.25,24.25,0,0,1,7.49-4.83,41.21,41.21,0,0,1,9.26-2.58l19.49-2.9q3.38-.48,4.67-2A5.45,5.45,0,0,0,679,96.69a9.56,9.56,0,0,0-3.3-7.25q-3.3-3.06-10.07-3.06-7.09,0-10.87,3.86a14.53,14.53,0,0,0-4.27,9l-19-4a26.1,26.1,0,0,1,2.9-9.34,28.81,28.81,0,0,1,6.6-8.38,33.31,33.31,0,0,1,10.31-6,39.64,39.64,0,0,1,14.17-2.34,45.18,45.18,0,0,1,15.94,2.5,29.79,29.79,0,0,1,10.71,6.6,24.5,24.5,0,0,1,6,9.58A35.44,35.44,0,0,1,700,99.27v39q0,3.22.32,6.84t.65,5.72H681.29a41.36,41.36,0,0,1-.56-4.43q-.24-2.82-.24-5.07a27.63,27.63,0,0,1-8.78,8.21q-5.72,3.54-14.57,3.54a30.56,30.56,0,0,1-11.36-2,27.16,27.16,0,0,1-8.62-5.31,22.52,22.52,0,0,1-5.39-7.57A22.2,22.2,0,0,1,629.92,129.22Zm31.73,7.89a23.87,23.87,0,0,0,6.52-.88,13.82,13.82,0,0,0,5.56-3.06,15.48,15.48,0,0,0,3.86-5.64,22.88,22.88,0,0,0,1.45-8.78v-3.54l-17.88,2.74a14.1,14.1,0,0,0-7,3q-2.82,2.33-2.82,7a9.18,9.18,0,0,0,2.58,6.36Q656.49,137.11,661.65,137.11Z"/><path class="cls-1" d="M790.14,136.47q0,4,.16,8t.48,6.36H770.33a27.47,27.47,0,0,1-.56-3.86q-.24-2.58-.24-5.32a22,22,0,0,1-8.46,7.89,27.64,27.64,0,0,1-13.77,3.22,38.15,38.15,0,0,1-15.62-3.14,36,36,0,0,1-12.16-8.7,40.78,40.78,0,0,1-8-13.21,48.86,48.86,0,0,1-.08-32.93,40.57,40.57,0,0,1,7.81-13.13,38.14,38.14,0,0,1,36.32-11A29.56,29.56,0,0,1,762.12,73a18.41,18.41,0,0,1,4.43,3.14,13,13,0,0,1,2.5,3.3V34.2h21.1ZM730.23,111q0,10.63,5.56,16.59a19.36,19.36,0,0,0,27.94,0q5.64-6,5.64-16.75,0-10.63-5.64-16.35a19.64,19.64,0,0,0-27.78.08Q730.23,100.4,730.23,111Z"/><path class="cls-1" d="M801.11,142.48a8,8,0,0,1,2.35-5.81,7.73,7.73,0,0,1,5.76-2.41,8.22,8.22,0,0,1,8.22,8.22,7.73,7.73,0,0,1-2.41,5.76,8,8,0,0,1-5.81,2.35,8,8,0,0,1-8.1-8.1Z"/><path class="cls-1" d="M828.05,56.11a11.69,11.69,0,0,1,23.38,0,11.69,11.69,0,1,1-23.38,0Zm1.86,94.6V75.92h19.66V150.7Z"/><path class="cls-1" d="M859.82,113.24c0-21.21,14.86-39.17,39.48-39.17,24.77,0,39.63,18,39.63,39.17s-14.86,39.33-39.63,39.33C874.68,152.56,859.82,134.45,859.82,113.24Zm58.68,0c0-11.61-6.81-21.68-19.2-21.68-12.23,0-19,10.06-19,21.68,0,11.77,6.81,21.83,19,21.83C911.68,135.07,918.49,125,918.49,113.24Z"/><path class="cls-2" d="M985.25,58a6,6,0,0,1,6-5.85,5.89,5.89,0,0,1,6,5.85,6,6,0,0,1-6,6A6.12,6.12,0,0,1,985.25,58Zm2.15,89.7v-72h8v72Z"/><path class="cls-2" d="M1068.4,147.68v-49c0-13.95-7-18.15-17.51-18.15-9.28,0-18.85,5.85-23.49,12.45v54.75h-8v-72h8V86.56c5.26-6.26,15.63-12.67,26.15-12.67,14.73,0,22.85,7.16,22.85,23.4v50.38Z"/><path class="cls-2" d="M1094.14,138.08l4.65-5.7c4.65,5.85,13.2,10.65,23.4,10.65,12.15,0,19.35-6.3,19.35-14.7,0-19.8-45.75-8.55-45.75-34.65,0-11.1,9.45-20.1,25.8-20.1,12.45,0,20.7,4.8,25.95,10.65l-4.35,5.55c-4.2-5.7-12-9.6-21.6-9.6-11.25,0-18,5.7-18,13.2,0,18.15,45.6,7,45.6,34.95,0,11.7-9,21.3-27.15,21.3C1110.34,149.63,1101.34,146.18,1094.14,138.08Z"/><path class="cls-2" d="M1170.4,134.28V82.68h-12v-7h12v-20h8v20h15v7h-15v50.4c0,5.85,2.42,9.9,7.55,9.9a11.43,11.43,0,0,0,8.15-3.45l3,6c-2.87,2.7-6.34,4.5-12.38,4.5C1175.08,150,1170.4,144.18,1170.4,134.28Z"/><path class="cls-2" d="M1210.4,147.68v-72h8V88.13c6-8.1,14.13-13.95,24-13.95v8.55a15.89,15.89,0,0,0-3.83-.3c-7.21,0-16.93,6.6-20.17,12.9v52.35Z"/><path class="cls-2" d="M1307.4,147.68v-10.2c-5.86,6.6-15.63,12.45-26.15,12.45-14.73,0-22.85-6.9-22.85-23.25v-51h8v49.2c0,14.1,7,18,17.51,18,9.28,0,18.7-5.55,23.49-12V75.68h7v72Z"/><path class="cls-2" d="M1335,111.53c0-21.15,13.8-37.95,35.1-37.95,12.6,0,19.8,4.95,25.65,12.3l-5.4,4.8c-5.25-7.2-11.85-10-19.8-10-16.95,0-27.3,13.5-27.3,30.9s10.35,31,27.3,31c7.95,0,14.55-3,19.8-10.2l5.4,4.8c-5.85,7.5-13.05,12.45-25.65,12.45C1348.84,149.63,1335,132.68,1335,111.53Z"/><path class="cls-2" d="M1414.4,134.28V82.68h-12v-7h12v-20h8v20h15v7h-15v50.4c0,5.85,2.42,9.9,7.55,9.9a11.43,11.43,0,0,0,8.15-3.45l3,6c-2.87,2.7-6.34,4.5-12.38,4.5C1419.08,150,1414.4,144.18,1414.4,134.28Z"/><path class="cls-2" d="M1450.39,111.53c0-21.15,13.5-37.95,34.65-37.95,21.45,0,34.95,16.8,34.95,37.95s-13.5,38.1-34.95,38.1C1463.88,149.63,1450.39,132.68,1450.39,111.53Zm61.2,0c0-16-9.45-30.9-26.55-30.9-16.8,0-26.4,14.85-26.4,30.9s9.6,31,26.4,31C1502.13,142.58,1511.58,127.73,1511.58,111.53Z"/><path class="cls-2" d="M1540.4,147.68v-72h8V88.13c6-8.1,14.14-13.95,24-13.95v8.55a15.89,15.89,0,0,0-3.83-.3c-7.21,0-16.93,6.6-20.17,12.9v52.35Z"/><path class="cls-2" d="M1581.33,138.08l4.65-5.7c4.65,5.85,13.2,10.65,23.4,10.65,12.15,0,19.35-6.3,19.35-14.7,0-19.8-45.75-8.55-45.75-34.65,0-11.1,9.45-20.1,25.8-20.1,12.45,0,20.7,4.8,25.95,10.65l-4.35,5.55c-4.2-5.7-12-9.6-21.6-9.6-11.25,0-18,5.7-18,13.2,0,18.15,45.6,7,45.6,34.95,0,11.7-9,21.3-27.15,21.3C1597.53,149.63,1588.53,146.18,1581.33,138.08Z"/><path class="cls-3" d="M147.95,15l-7.74,4.62a.9.9,0,0,0,0,1.54l7.74,4.62a.9.9,0,0,1,.32,1.21l-4,7.12a.9.9,0,0,1-1.24.33l-7.63-4.56a.9.9,0,0,0-1.36.77v9.2a.9.9,0,0,1-.9.9H125a.9.9,0,0,1-.9-.9V30.64a.9.9,0,0,0-1.36-.77l-7.5,4.54a.9.9,0,0,1-1.24-.32L109.9,27a.9.9,0,0,1,.32-1.22l7.83-4.61a.9.9,0,0,0,0-1.55L110.22,15a.9.9,0,0,1-.32-1.22L114,6.63a.9.9,0,0,1,1.24-.32l7.5,4.54a.9.9,0,0,0,1.36-.77V.9A.9.9,0,0,1,125,0h8.12a.9.9,0,0,1,.9.9v9.2a.9.9,0,0,0,1.36.77L143,6.31a.9.9,0,0,1,1.24.33l4,7.12A.9.9,0,0,1,147.95,15Z"/><path class="cls-4" d="M46.75,18.32c31.93-8.67,55.87,25.95,69.91,58.23h-3.91l.81,14.89,7.65,14.51,5.4-2.11c.95,3.2,1.74,6.17,2.38,8.84,7.62,31.61-13,63.14-46.27,70.49S16.12,170.88,8.37,139.3,13.87,27.24,46.75,18.32Z"/><path class="cls-5" d="M130.06,118.4a57.12,57.12,0,0,0-1.07-5.73c-.09-.35-.18-.73-.27-1.09l-2.56,1.33-4.75-7-.2.08-.91-1.72-5.45-8.08-.54-3.29-.75-1.42-.25-4.64L111.86,78l1-.1-.08-1.4h3.32c-2.41-4.18-5-8.37-7.67-12.5-4.21,1.13-8.51,3-10.26,5.72-5.37,1.07-12.74-2.66-15-3.34-17.22-5.2-28.33-3.35-45.89.25a8.25,8.25,0,0,0-1.89.64l0-.1L9.82,81.5c-1.12,5.8-2,11.63-2.58,17.32l5-3.17s16.77,45,24.26,45.15c10.24.21,34.57-3.11,43-9.89C91,121.72,94.49,117.07,94.1,105.58c.18-6.91,0-12.71,2.47-13.06S104.38,129.84,130.06,118.4Zm-91.89,9.95c-6.11-10.22-17.47-37.44-17.48-37.84.14,0,21.51-17.39,52.06-14.16C83,138.19,38.43,128.8,38.17,128.35Z"/><polygon class="cls-4" points="125.59 64.83 132.76 108.01 121.16 107.82 115.07 72.21 115.48 64.96 125.59 64.83"/><path class="cls-6" d="M134.1,103.94a15.71,15.71,0,0,1-4.78,1.89l-5.24.55a15.71,15.71,0,0,0,4.78-1.89Z"/><path class="cls-6" d="M61.51,65.41c-6.71,0-17.09,4.46-16.9,4.44,12.7-1.61,24.29-2.62,31.66,0l5.44-1.2C74.35,66,68.79,65.46,61.51,65.41Z"/><path class="cls-6" d="M123.39,67.49l5.29-.09c.08.49,5.37,35.69,5.42,36.53l-5.24.55C128.81,103.65,123.47,68,123.39,67.49Z"/><path class="cls-7" d="M123.44,67.83a32,32,0,0,0-6.81-1l3.23-.34a26.22,26.22,0,0,1,8.82.94Z"/><path class="cls-6" d="M106.83,69c-.07-.46-.11-.91-.15-1.35v.08c3.64,24.35,3.91,43.17,20.07,48.6a8.76,8.76,0,0,0,3.69.31l-4.77.5C110.73,111.22,110.38,92.74,106.83,69Z"/><path class="cls-8" d="M101.44,68.24c-.36-4.57,1.49-6.71,3.38-8.05a17.51,17.51,0,0,1,4.81-2.41c-3.18,1.75-3.54,5.8-2.95,9.9Z"/><path class="cls-6" d="M106.68,67.68c0,.44.08.88.15,1.35,3.55,23.72,3.9,42.19,18.86,48.14l-.48.05a8.76,8.76,0,0,1-3.69-.31c-16.16-5.42-16.43-24.25-20.07-48.6v-.07Z"/><path class="cls-6" d="M105.86,96.09s-1.85-9.67-6.46-8.52c-2.84.71-1.5,4.38-2,12.94L88.53,67.57c10.17,5.15,11.92,1.21,18.61-1.33Z"/><path class="cls-6" d="M76.26,69.76c12.27,48.9-3.39,53.5-22.53,55.92l5.67-.6c35-4.42,27.14-26.24,22.72-55.08a4.86,4.86,0,0,0-.41-1.39s-5.52.92-5.45,1.15"/><path class="cls-9" d="M48.32,135.21l-5.67.6c1.37-.06,3-.18,4.71-.36l5.67-.6C51.28,135,49.69,135.14,48.32,135.21Z"/><path class="cls-6" d="M41.69,125.07l-5.67.6c1.11,7.25,5.82,12.2,10.7,11.68l5.67-.6C47.51,137.27,42.8,132.32,41.69,125.07Z"/><path class="cls-6" d="M0,96S2.29,84.19,5.88,82.12L43.8,61l-4,15.5-5.17,2.76,7,45.84-5.67.6L30.15,87.31a3.86,3.86,0,0,0-5.68-2.63L7.32,96.76Z"/><path class="cls-10" d="M0,96,26.27,77.1C30.81,74,34.64,79,35.08,81.93l6.61,43.14c1.11,7.25,5.82,12.2,10.7,11.68a6.78,6.78,0,0,0,1.06-.2c1.37-.06,3-.18,4.71-.36,8.19-.87,19.53-3.18,25.23-7.75,12.78-10.24,15-20.64,13.75-34-.59-14.49,3-16.23,5.94-15.71s5,7.85,6,12.21c2.4,12.75,6.27,22,17.14,25.48a9.32,9.32,0,0,0,3.81.29c4.11-.43,6-1.57,8.82-3.55,2.71-1.87,3.34-5.38,2.55-10.59-5.11-33.86-5.19-42.43-10.28-43.7-6.54-1.64-10.21-3.09-22.38-1-3.13,1.2-7.67,9-18.77,3C79.65,53,47.17,55.54,36.46,58.3L5.94,79.78C2.36,81.85.2,87.48,0,91.45ZM116.3,67.07c3.89-1,7-.78,12.09.37.08.49,5.42,35.68,5.46,36.53a16.67,16.67,0,0,1-4.95,1.91C120.1,102.61,119.92,91,116.3,67.07ZM44.61,69.85C58.79,67,68.09,65.5,81.71,68.61A4.85,4.85,0,0,1,82.11,70c4.41,28.84,12.24,50.66-22.72,55.08a6.38,6.38,0,0,1-7.09-5.17C49.91,105.73,44.61,70.25,44.61,69.85Z"/><path class="cls-10" d="M60,64.52h0Zm0,0h0Z"/></g></g></svg>
