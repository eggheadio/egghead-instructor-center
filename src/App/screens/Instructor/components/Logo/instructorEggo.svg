<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 148.39 184.71"><defs><style>.cls-1{fill:url(#linear-gradient);}.cls-2{fill:#f1edeb;}.cls-3{fill:#9399a1;opacity:0.3;}.cls-4{fill:#e97c43;}.cls-5{fill:#ec8030;}.cls-6{fill:#f09f4f;}.cls-7{fill:#cbcc5f;}.cls-8{fill:#fbd170;}</style><linearGradient id="linear-gradient" x1="10477.89" y1="30.38" x2="10477.89" y2="65.46" gradientTransform="matrix(-1, 0, 0, 1, 10606.98, 0)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#fcd06a"/><stop offset="1" stop-color="#ee8d4c"/></linearGradient></defs><title>Teacher-Eggo-onDark</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M147.95,15l-7.74,4.62a.9.9,0,0,0,0,1.54l7.74,4.62a.9.9,0,0,1,.32,1.21l-4,7.12a.9.9,0,0,1-1.24.33l-7.63-4.56a.9.9,0,0,0-1.36.77v9.2a.9.9,0,0,1-.9.9H125a.9.9,0,0,1-.9-.9V30.64a.9.9,0,0,0-1.36-.77l-7.5,4.54a.9.9,0,0,1-1.24-.32L109.9,27a.9.9,0,0,1,.32-1.22l7.83-4.61a.9.9,0,0,0,0-1.55L110.22,15a.9.9,0,0,1-.32-1.22L114,6.63a.9.9,0,0,1,1.24-.32l7.5,4.54a.9.9,0,0,0,1.36-.77V.9A.9.9,0,0,1,125,0h8.12a.9.9,0,0,1,.9.9v9.2a.9.9,0,0,0,1.36.77L143,6.31a.9.9,0,0,1,1.24.33l4,7.12A.9.9,0,0,1,147.95,15Z"/><path class="cls-2" d="M46.75,18.32c31.93-8.67,55.87,25.95,69.91,58.23h-3.91l.81,14.89,7.65,14.51,5.4-2.11c.95,3.2,1.74,6.17,2.38,8.84,7.62,31.61-13,63.14-46.27,70.49S16.12,170.88,8.37,139.3,13.87,27.24,46.75,18.32Z"/><path class="cls-3" d="M130.06,118.4a57.12,57.12,0,0,0-1.07-5.73c-.09-.35-.18-.73-.27-1.09l-2.56,1.33-4.75-7-.2.08-.91-1.72-5.45-8.08-.54-3.29-.75-1.42-.25-4.64L111.86,78l1-.1-.08-1.4h3.32c-2.41-4.18-5-8.37-7.67-12.5-4.21,1.13-8.51,3-10.26,5.72-5.37,1.07-12.74-2.66-15-3.34-17.22-5.2-28.33-3.35-45.89.25a8.25,8.25,0,0,0-1.89.64l0-.1L9.82,81.5c-1.12,5.8-2,11.63-2.58,17.32l5-3.17s16.77,45,24.26,45.15c10.24.21,34.57-3.11,43-9.89C91,121.72,94.49,117.07,94.1,105.58c.18-6.91,0-12.71,2.47-13.06S104.38,129.84,130.06,118.4Zm-91.89,9.95c-6.11-10.22-17.47-37.44-17.48-37.84.14,0,21.51-17.39,52.06-14.16C83,138.19,38.43,128.8,38.17,128.35Z"/><polygon class="cls-2" points="125.59 64.83 132.76 108.01 121.16 107.82 115.07 72.21 115.48 64.96 125.59 64.83"/><path class="cls-4" d="M134.1,103.94a15.71,15.71,0,0,1-4.78,1.89l-5.24.55a15.71,15.71,0,0,0,4.78-1.89Z"/><path class="cls-4" d="M61.51,65.41c-6.71,0-17.09,4.46-16.9,4.44,12.7-1.61,24.29-2.62,31.66,0l5.44-1.2C74.35,66,68.79,65.46,61.51,65.41Z"/><path class="cls-4" d="M123.39,67.49l5.29-.09c.08.49,5.37,35.69,5.42,36.53l-5.24.55C128.81,103.65,123.47,68,123.39,67.49Z"/><path class="cls-5" d="M123.44,67.83a32,32,0,0,0-6.81-1l3.23-.34a26.22,26.22,0,0,1,8.82.94Z"/><path class="cls-4" d="M106.83,69c-.07-.46-.11-.91-.15-1.35v.08c3.64,24.35,3.91,43.17,20.07,48.6a8.76,8.76,0,0,0,3.69.31l-4.77.5C110.73,111.22,110.38,92.74,106.83,69Z"/><path class="cls-6" d="M101.44,68.24c-.36-4.57,1.49-6.71,3.38-8.05a17.51,17.51,0,0,1,4.81-2.41c-3.18,1.75-3.54,5.8-2.95,9.9Z"/><path class="cls-4" d="M106.68,67.68c0,.44.08.88.15,1.35,3.55,23.72,3.9,42.19,18.86,48.14l-.48.05a8.76,8.76,0,0,1-3.69-.31c-16.16-5.42-16.43-24.25-20.07-48.6v-.07Z"/><path class="cls-4" d="M105.86,96.09s-1.85-9.67-6.46-8.52c-2.84.71-1.5,4.38-2,12.94L88.53,67.57c10.17,5.15,11.92,1.21,18.61-1.33Z"/><path class="cls-4" d="M76.26,69.76c12.27,48.9-3.39,53.5-22.53,55.92l5.67-.6c35-4.42,27.14-26.24,22.72-55.08a4.86,4.86,0,0,0-.41-1.39s-5.52.92-5.45,1.15"/><path class="cls-7" d="M48.32,135.21l-5.67.6c1.37-.06,3-.18,4.71-.36l5.67-.6C51.28,135,49.69,135.14,48.32,135.21Z"/><path class="cls-4" d="M41.69,125.07l-5.67.6c1.11,7.25,5.82,12.2,10.7,11.68l5.67-.6C47.51,137.27,42.8,132.32,41.69,125.07Z"/><path class="cls-4" d="M0,96S2.29,84.19,5.88,82.12L43.8,61l-4,15.5-5.17,2.76,7,45.84-5.67.6L30.15,87.31a3.86,3.86,0,0,0-5.68-2.63L7.32,96.76Z"/><path class="cls-8" d="M0,96,26.27,77.1C30.81,74,34.64,79,35.08,81.93l6.61,43.14c1.11,7.25,5.82,12.2,10.7,11.68a6.78,6.78,0,0,0,1.06-.2c1.37-.06,3-.18,4.71-.36,8.19-.87,19.53-3.18,25.23-7.75,12.78-10.24,15-20.64,13.75-34-.59-14.49,3-16.23,5.94-15.71s5,7.85,6,12.21c2.4,12.75,6.27,22,17.14,25.48a9.32,9.32,0,0,0,3.81.29c4.11-.43,6-1.57,8.82-3.55,2.71-1.87,3.34-5.38,2.55-10.59-5.11-33.86-5.19-42.43-10.28-43.7-6.54-1.64-10.21-3.09-22.38-1-3.13,1.2-7.67,9-18.77,3C79.65,53,47.17,55.54,36.46,58.3L5.94,79.78C2.36,81.85.2,87.48,0,91.45ZM116.3,67.07c3.89-1,7-.78,12.09.37.08.49,5.42,35.68,5.46,36.53a16.67,16.67,0,0,1-4.95,1.91C120.1,102.61,119.92,91,116.3,67.07ZM44.61,69.85C58.79,67,68.09,65.5,81.71,68.61A4.85,4.85,0,0,1,82.11,70c4.41,28.84,12.24,50.66-22.72,55.08a6.38,6.38,0,0,1-7.09-5.17C49.91,105.73,44.61,70.25,44.61,69.85Z"/><path class="cls-8" d="M60,64.52h0Zm0,0h0Z"/></g></g></svg>
