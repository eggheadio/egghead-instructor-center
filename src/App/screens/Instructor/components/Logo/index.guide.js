import React from 'react'
import {storiesOf} from '@kadira/storybook'
import Logo from '.'

storiesOf('Logo', module)
  .add('default', () => (
    <Logo />
  ))
