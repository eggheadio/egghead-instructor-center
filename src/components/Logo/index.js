import React from 'react'
import eggoIcon from './eggoIcon.svg'
import eggoInstructorBanner from './eggoInstructorBanner.svg'
import instructorBanner from './instructorBanner.svg'

export const EggoIcon = ({className}) => (
  <img
    src={eggoIcon}
    alt='egghead.io instructors logo, eggo only'
    className={className}
  />
)

export const EggoInstructorBanner = ({className}) => (
  <img
    src={eggoInstructorBanner}
    alt='egghead.io instructors logo'
    className={className}
  />
)

export const InstructorBanner = ({className}) => (
  <img
    src={instructorBanner}
    alt='egghead.io instructors logo'
    className={className}
  />
)
