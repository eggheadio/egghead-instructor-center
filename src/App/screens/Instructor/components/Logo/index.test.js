import React from 'react'
import {create} from 'react-test-renderer'
import Logo from '.'

test('default', () => (
  expect(create(
    <Logo />
  )).toMatchSnapshot()
))
