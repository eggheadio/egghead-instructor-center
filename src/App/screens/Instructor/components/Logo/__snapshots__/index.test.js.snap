exports[`test default 1`] = `
<img
  alt="egghead.io logo"
  src="logo.svg" />
`;
